api/src/test/java/org/apache/iceberg/transforms/TestProjection.java

```diff
@@ -216,15 +219,15 @@ public void testCaseInsensitiveStrictIdentityProjection() {
 
   @Test
   public void testCaseSensitiveStrictIdentityProjection() {
-    PartitionSpec spec = PartitionSpec.builderFor(SCHEMA)
-      .identity("id")
-      .build();
+    PartitionSpec spec = PartitionSpec.builderFor(SCHEMA).identity("id").build();
```

Comment:
Is the rule being applied here "If it fits in 120 chars, don't allow line breaks"? Or was this a manual change?

Reply:
Nah this was a manual change, it's not for any particular rule. We can revert.

Reply:
I like keeping these on the same line if they will fit, but I don't feel strongly about it.